edit-post/components/layout/index.js

```diff
@@ -81,8 +82,9 @@ function Layout( {
 				/>
 			) }
 			{ editorSidebarOpened && <Sidebar /> }
-			{ pluginSidebarOpened && <PluginsPanel /> }
+			{ pluginSidebarOpened && <PluginSidebar /> }
 			<Popover.Slot />
+			<PluginArea />
```

Comment:
If this is truly "PluginArea", why is it specific to edit-post ? Why is it imported as PluginSidebar? Isn't the general pattern that we'd render this area once on every screen?